src/main/java/io/phasetwo/keycloak/magic/MagicLink.java

```diff
@@ -99,7 +99,8 @@ public static MagicLinkActionToken createActionToken(
       String clientId,
       OptionalInt validity,
       Boolean rememberMe,
-      AuthenticationSessionModel authSession) {
+      AuthenticationSessionModel authSession,
+      Boolean isActionTokenPersistent) {
```

Comment:
Since this method is public, it should not be altered. Make a new method with the new arguments, and update this method to call the new one with the the default (true).

Reply:
Any particular reason why? These functions seem to only have one usage, injection point from (MagicLinkAuthenticator, MagicLinkResource). Those injection point is where the state of `isActionTokenPersistent` comes from.

Reply:
Made suggested changes